js.js

```diff
@@ -0,0 +1,124 @@
+console.log("hello world!");
```

Comment:
Can you split up your homework in separate files (as was asked in the assignment)?

```
For all the following exercises create a new .js file. Try to find a proper name for each file or make a small comment about what it does inside for future reference.
```

Reply:
yup ...I'm really sorry ! 
i didn't notice that..